python/smqtk/utils/postgres.py

```diff
@@ -87,18 +91,32 @@ def __init__(self, db_name='postgres', db_host=None, db_port=None,
         self.table_upsert_lock = table_upsert_lock
         self.table_upsert_sql = None
 
+        key_tuple = (
+            self.db_name,
+            self.db_user,
+            self.db_pass,
+            self.db_host,
+            self.db_port
+        )
+        try:
+            self.connection_pool = _connection_pools[key_tuple]
+        except KeyError:
+            _connection_pools[key_tuple] = self.connection_pool = \
+                psycopg2.pool.ThreadedConnectionPool(
+                    0, 100,
```

Comment:
Expose the min/max connection parameters as defaulted parameters in the ``PsqlConnectionHelper`` constructor.

Reply:
What happens if two different calling functions create a `PsqlConnectionHelper` with different min and max params? Then the params are dictated by whichever one gets there first. Is there another, better way to do it? Some sort of configuration file?

Reply:
Valid corner case.  I'm just thinking that different applications will want to set different pool ``maxconn`` limits (less likely the ``minconn`` param).  Maybe for now just comment that we are hard-coding the min/max-conn values at what we think are currently sensible values.